evaluator/src/test/kotlin/ProjectSourceRuleTest.kt

```diff
@@ -142,3 +177,53 @@ private fun File.addDirs(vararg paths: String) {
         resolve(path).mkdirs()
     }
 }
+
+private fun ortResultWithDetectedLicenses(vararg detectedLicensesForFilePath: Pair<String, Set<String>>): OrtResult {
+    val id = Identifier("Maven:org.oss-review-toolkit:example:1.0")
+    val vcsInfo = VcsInfo(
+        type = VcsType.GIT,
+        url = "https://github.com/oss-review-toolkit/example.git",
+        revision = "0000000011111111000000001111111100000000"
+    )
+    val licenseFindings = detectedLicensesForFilePath.flatMapTo(sortedSetOf()) { (filepath, licenses) ->
+        licenses.map { license ->
+            LicenseFinding(license, TextLocation(filepath, startLine = 1, endLine = 2),)
+        }
+    }
+
+    return OrtResult.EMPTY.copy(
+        repository = Repository(vcsInfo),
+        analyzer = AnalyzerRun(
+            config = AnalyzerConfiguration(),
+            environment = Environment(),
+            result = AnalyzerResult.EMPTY.copy(
+                projects = sortedSetOf(
+                    Project.EMPTY.copy(
+                        id = id,
+                        vcsProcessed = vcsInfo
+                    )
+                )
+            )
+        ),
+        scanner = ScannerRun.EMPTY.copy(
+            results = ScanRecord(
+                scanResults = sortedMapOf(
+                    id to listOf(
+                        ScanResult(
+                            provenance = RepositoryProvenance(vcsInfo, vcsInfo.revision),
+                            scanner = ScannerDetails.EMPTY,
+                            summary = ScanSummary(
+                                licenseFindings = licenseFindings,
+                                copyrightFindings = sortedSetOf(),
+                                startTime = Instant.EPOCH,
+                                endTime = Instant.EPOCH,
+                                packageVerificationCode = ""
```

Comment:
We currently use a mix of empty strings, all zeroes, and `SpdxConstants.NONE` for dummy `packageVerificationCode`s. Given that the `packageVerificationCode` also is a SHA1, I have a tendency towards also using all zeros here (in this PR, I mean; I can do an alignment commit for the whole code base afterwards).